chrobalt/build/configs/variables.gni

```diff
@@ -0,0 +1,47 @@
+# Copyright 2024 The Cobalt Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Get the path to the starboard implementation and include its GN
+# configuration.
+
+
+import("//$starboard_path/platform_configuration/configuration.gni")
+
+sb_is_modular = sb_is_evergreen || build_with_separate_cobalt_toolchain
+
+declare_args() {
+  use_tsan = getenv("USE_TSAN") == 1
```

Comment:
Needed ? Unless we have build configs defined for it i would cut this out

Reply:
removed